extension/js/common/core/pgp-password.ts

```diff
@@ -0,0 +1,40 @@
+/* © 2016-2018 FlowCrypt Limited. Limitations apply. Contact human@flowcrypt.com */
+
+'use strict';
+
+import { Pgp, openpgp } from './pgp.js';
+import { base64encode, secureRandomBytes } from '../platform/util.js';
+
+interface PasswordStrengthResult {
+  word: {
+    match: string;
+    word: string;
+    bar: number;
+    color: string;
+    pass: boolean;
+  };
+  seconds: number;
+  time: string;
+}
+
+export class PgpPwd {
+  static estimateStrength = (zxcvbnResultGuesses: number, type: 'passphrase' | 'pwd' = 'passphrase'): PasswordStrengthResult => {
+    const timeToCrack = zxcvbnResultGuesses / Pgp.CRACK_GUESSES_PER_SECOND;
+    for (const word of type === 'pwd' ? Pgp.CRACK_TIME_WORDS_PWD : Pgp.CRACK_TIME_WORDS_PASS_PHRASE) {
```

Comment:
please move `CRACK*` props to this class

Reply:
Then you can probably make them private

Reply:
I didn't notice that,
Thanks